mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -834,6 +1009,7 @@
         <CardLayouts index="0">
             <LayoutButtons index="10" removed="1"/>
         </CardLayouts>
+        <Sight value="12"/>
```

Comment:
Lair sight range increased from 11 to 12 to reach to the end of its own creep spread